cli/serverless/wasm/runtime.go

```diff
@@ -3,15 +3,19 @@ package wasm
 
 import (
 	"fmt"
+
+	"github.com/yomorun/yomo/serverless"
 )
 
 // Define wasm import/export function names
 const (
+	WasmFuncStart          = "_start"
 	WasmFuncInit           = "yomo_init"
 	WasmFuncObserveDataTag = "yomo_observe_datatag"
-	WasmFuncLoadInput      = "yomo_load_input"
-	WasmFuncDumpOutput     = "yomo_dump_output"
```

Comment:
If these export functions were redefined, examples of other languages should also be modified, and docs as well.

Reply:
Yeah, we will introduce a new PR for examples and documents.